analyses/cell-type-wilms-tumor-06/notebook_template/03_clustering_exploration.Rmd

```diff
@@ -0,0 +1,863 @@
+---
+title: "Clustering exploration for `r params$sample_id`"
+author: "Maud PLASCHKA"
+date: '2024-08-13'
+params:
+  scpca_project_id: "SCPCP000006"
+  sample_id: "SCPCS000176"
+  seed: 12345
+  padj_thershold: 0.05  # Threshold for adjusted p-value
+  lfc_threshold: 1      # Threshold for log fold change
+  rate1_threshold: 0.5  # Threshold for rate (percentage) of cells expressing the marker genes in group1
+output: 
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      message=FALSE,
+                      warnings=FALSE
+                      )
+```
+
+
+## Introduction
+
+The aim is to explore the clustering and label transfers for the sample `r params$sample_id`.
+
```

Comment:
This suggestion goes along with a recommendation I made in my overall review comment.

```suggestion
This sample has a(n) `r favorable` subdiagnosis.
```